go/vt/vitessdriver/doc.go

```diff
@@ -0,0 +1,114 @@
+// Package vitessdriver contains the Vitess Go SQL driver.
+/*
+Vitess
+
+Vitess is an SQL middleware which turns MySQL/MariaDB into a fast, scalable and
+highly-available distributed database.
+For more information, visit http://www.vitess.io.
+
+
+Example
+
+Using this SQL driver is as simple as:
+
+  import (
+    "time"
+    "github.com/youtube/vitess/go/vt/vitessdriver"
+  )
+
+  func main() {
+    // Connect to vtgate (v1 mode).
+    db, err := vitessdriver.Open("localhost:15991", "keyspace", "0", "master", 30*time.Second)
+
+    // Use "db" via the Golang sql interface.
+  }
+
+For a full example, please see: https://github.com/youtube/vitess/blob/master/examples/local/client.go
+
+The full example is based on our tutorial for running Vitess locally: http://vitess.io/getting-started/local-instance.html
+
+
+Vtgate
```

Comment:
Since vtgate is always lowercase everywhere else, it should probably be lowercase in headers also.